org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -22,51 +22,51 @@ <h2>Snapshot Build @qualified.bundle.version@ (@build.date@)</h2>
 
 <h3>New Features</h3>
 <ul>
-  <li>Limit configuration of Maven check goal and Ant check element now also
-      supports percentage values 
+  <li>JaCoCo now comes with a simple command line interface
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/525">#525</a>).</li>
+  <li>Limit configuration of Maven <code>check</code> goal and Ant
+      <code>check</code> element now also supports percentage values 
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/534">#534</a>).</li>
-  <li>Exclude from a report enum methods <code>valueOf</code> and <code>values</code>
-      that are created by compiler
+  <li>Manifests of JAR files now have <code>Automatic-Module-Name</code> for Java 9
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/565">#565</a>).</li>
+  <li>Maven aggregated reports will now also include modules of <code>runtime</code>
+      and <code>provided</code> dependencies
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/498">#498</a>,
+      <a href="https://github.com/jacoco/jacoco/issues/572">#572</a>).</li>
+</ul>
+
+<p>
+  JaCoCo reports now filter out various compiler generated artifacts which
+  would lead to partial or missed coverage:
```

Comment:
what about
```
During creation of reports various compiler generated artifacts are filtered out,
which otherwise require unnecessary and sometimes impossible tricks to not have partial or missed coverage:
```
in order to clearly (hopefully) state that this happens after consumption of exec-file, and to clarify a bit more about which kinds of partial/missed coverage?

Reply:
btw not sure that Lombok can be named compiler

Reply:
Inserted you text. Sure Lombok is not a compiler. Just wondering whether it is worth to create a separate section for it?

Reply:
@marchof after reading recent message in mailing list, not sure that this rephrasing will help as people do not understand difference between exec-file and report 😞  even not sure that addition of explicit `(not during creation of exec-file)` will help 😆 

and so let's leave also Lombok as is.